src/components/snapshots/useSnapshot.ts

```diff
@@ -59,6 +63,9 @@ const useSnapshot: SnapshotOperation = async ({
       throw `Snapshot was created with version: ${snapshotVersion}\n You can't install a snapshot created with a version newer than the current application version: ${config.version}`
     }
 
+    // Check that we can find all the archives needed:
+    await collectArchives(snapshotFolder)
```

Comment:
The called function `findArchiveSources` throws an error if it can't find all the archives required to construct the full snapshot, so this won't proceed any further.